roles/migrid_logstash_gdp/tasks/logstash.yml

```diff
@@ -0,0 +1,91 @@
+---
+# role for installing logstash for consuming the gdp.log file. 
+# We want to be sure that no data is lost during logfile consuming, and that everything ends in loki
+# requires:
+#   setup_vars
+#   community.general (.logstash_plugin) version 7.5.0 or greater
+#   promtail_migrid   requires that user that you defined for logscraping on lustre mds is defined on server
+
+#import promtail user task to ensure the user used here is available (reusing the promtail user for lustre access)
+- name: Import user create task
+  import_tasks:
+    file: ../../promtail_migrid/tasks/user.yml
+
+- name: Ensure Elastic keys are available
```

Comment:
This and the following tasks (everything thats apt related) should have `when: ansible_fact['os_family'] == "Debian" or similar.

Another option is to put them in a seperate file named `install-Debian.yml` and include them with

```
import_task:
   name: install-{{ ansible_facts['os_family'] }}.yml
```

or something similar